src/client/opamCommands.ml

```diff
@@ -2930,14 +2930,16 @@ let clean =
         OpamConsole.msg "rm -rf \"%s\"/*\n"
           (OpamFilename.Dir.to_string d)
       else
-        OpamFilename.cleandir d
+      try OpamFilename.cleandir d
+      with OpamSystem.Internal_error msg -> OpamConsole.note "Error ignored: %s" msg
```

Comment:
I'd prefer warning instead of simple notes

Reply:
Fair enough, fixed